File: Baekjoon/Greedy/17509 And the Winner Is... Ourselves!/HwangInGyu.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;

class Main {
    public static void main(String[] args) throws IOException {
        int numOfProplems = 11;
        int[] penalties = new int[numOfProplems];
        int wrongCount = 0;
        BufferedReader bfr = new BufferedReader(new InputStreamReader(System.in));
        for (int i = 0;  i < numOfProplems; i++) {
            String[] line = bfr.readLine().split(" ");
            penalties[i] = Integer.parseInt(line[0]);
            wrongCount += Integer.parseInt(line[1]);
        }
        Arrays.sort(penalties);

        int answer = 0;
        int sum = 0;
        for (int i = 0; i < numOfProplems; i++) {
            sum += penalties[i];
            answer += sum;
        }

        System.out.println(answer + wrongCount * 20);
    }
}

File: Baekjoon/완전탐색(Brute Force)/10797 10부제/HwangInGyu.kt
import java.io.BufferedReader
import java.io.InputStreamReader


fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    var cntOfRuleBreaker = 0
    val numberOfRestCar = readLine().toInt()
    readLine().split(" ").map {
        if (it.toInt() == numberOfRestCar) {
            cntOfRuleBreaker++
        }
    }
    println(cntOfRuleBreaker)
}
